test/connection.test.ts

```diff
@@ -523,6 +523,50 @@ describe('Connection', function () {
       assert.calledOnce(spy)
       assert.calledWith(spy, '58')
     })
+
+    it('should not send money from server in receiveOnly mode', async function () {
```

Comment:
Does this test that money is not sent? 

It seems to me it tests that the connection can be established without the server sending test money packets.